requests-for-comments/2023-01-12-plan-naming.md

```diff
@@ -0,0 +1,47 @@
+# Request for comments: Renaming our plans by @raquelmsmith
+
+## Problem statement
+_Who are we building for, what are their needs (include example cases), why is this important?_
+
+Plan names are something seemingly minor that actually get used quite a lot, both internally and by our customers (oftentimes just inside their own heads when evaluating). However, our current plan names leave room for confusion:
+
+* The names don't indicate right at the get-go which plan you are probably interested in based on your company size/scale/needs. 
+* The names can include information about deployment but lack any real plan name (eg. PostHog Cloud is a paid tier but it's also a deployment)
+* We use the term self-serve in the docs but no one really knows what this means
+
+## Proposal
+The Cloud environment was new a couple years ago, so it made sense to call the plans on Cloud the "Cloud" plans. However, the Cloud environment is by far the default way to use PostHog now, and calling the plans "Cloud" is confusing (I can imagine thinking, "should I be here? or is there a different non-cloud thing I should use?").
+
+So, I'd propose that our Cloud plans are the _default_ plans and are named:
+
+- **Starter**
+  - I prefer Starter over "Free" because "free" can get confused with the FOSS version
+  - I prefer Starter over "Hobby" because someone might be getting a new startup up and running, and they are very serious about it and working full time on it and are definitely not considering themselves a hobbyist, but this is still the correct plan for them
+- **Scale**
+  - I think this speaks to our goal to have post-PMF companies who are wanting to grow quickly be on paid plans
+- **Enterprise**
+
+Since we are sunsetting the k8s deployment, I don't think we will be selling licenses for self-hosted versions any more, correct? If that is the case then the only other option is the self-hosted docker deployment. We could simply call this the **Open-source** plan.
```

Comment:
We are, but at the 5k$ enterprise tier.

Reply:
Are you sure? https://github.com/PostHog/posthog.com/pull/5073